io/disk/ssd/nvmetest.py

```diff
@@ -345,9 +351,13 @@ def test_create_full_capacity_ns(self):
         """
         Test to create namespace with full capacity
         """
-        self.delete_all_ns()
-        self.create_full_capacity_ns()
-        self.list_ns()
+        device = self.device.split("/")[-1]
+        nvme.delete_all_ns(device)
+        nvme.create_full_capacity_ns(device)
+        if self.shared:
+            nvme.delete_all_ns(device)
+            nvme.create_full_capacity_ns(device,
+                                         shared_ns=self.shared)
```

Comment:
nvme.delete_all_ns(device) looks duplicate ?

Reply:
also self.shared can have default value as False

no need to add duplicate function call inside check... just have the flag to avoid redundant function calls

Reply:
ok covering normal namespace and shared names space on splitter

Reply:
@Naresh-ibm your input here :

what my ask is, no need of if condition here.. and self.shared flag can be passed from yaml.. for normal and splitter... and it can use same  old code with just a flag.... and when splitter ..by covering shared name space creation.. than no need to test normal name space ?

 nvme.delete_all_ns(device)
 nvme.create_full_capacity_ns(device, shared_ns=self.shared)